src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -48,10 +49,24 @@ class ReportActionItemFragment extends React.PureComponent {
                 }
 
                 // Only render HTML if we have html in the fragment
-                return fragment.html !== fragment.text ? (
-                    <RenderHTML html={fragment.html} debug={false} />
-                ) : (
-                    <Text selectable>{Str.htmlDecode(fragment.text)}</Text>
+                return (
+                    <View style={styles.flexRow}>
+                        {fragment.html !== fragment.text ? (
+                            <RenderHTML html={fragment.html} debug={false} />
+                        ) : (
+                            <Text selectable>{Str.htmlDecode(fragment.text)}</Text>
+                        )}
+                        {fragment.isEdited
+                            ? (
```

Comment:
Instead of:

```jsx
{fragment.isEdited
    ? (
        <Text />
    ) : null}
```

Let's just do:

```jsx
{fragment.isEdited && (
    <Text />
)}
```